roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,54 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdVolume(OpenShiftCheck):
+    """Ensures etcd storage usage does not exceed a given threshold."""
+
+    name = "etcd_volume"
+    tags = ["etcd", "health"]
+
+    default_etcd_device_usage_threshold_percent = 90
+    # where to find ectd data, higher priority first.
+    supported_mount_paths = ["/var/lib/etcd", "/var/lib", "/var", "/"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        # TODO: only execute this check on hosts in the 'ectd' group?
+        # Maybe also 'masters' if there are no standalone etcd hosts?
```

Comment:
@juanvallejo @sosiouxme what do you guys think? Originally the docstring of the class hinted that this was only run on "etcd hosts"...